packages/runtime/src/helpers/edge.ts

```diff
@@ -300,11 +305,13 @@ export const writeRscDataEdgeFunction = async ({
       function: 'rsc-data',
       name: 'RSC data routing',
       path,
+      generator: `${nextjsPluginVersion}`,
```

Comment:
This doesn't need to be a template string now and you can just use the variable.

```suggestion
      generator: nextjsPluginVersion`,
```